tests/tracing/test_provider.py

```diff
@@ -39,6 +44,7 @@ def test_span_processor_and_exporter_default():
 
 def test_span_processor_and_exporter_model_serving(monkeypatch):
     monkeypatch.setenv("IS_IN_DB_MODEL_SERVING_ENV", "True")
+    monkeypatch.setenv("ENABLE_MLFLOW_TRACING", "true")
```

Comment:
There are a few more tracing tests that simulate model serving env (search with `IS_IN_DB_MODEL_SERVING_ENV`). Could you update them too?